### PR TITLE
Adding documentation for flume notifications

### PR DESCRIPTION
**Pull request by jeeftor**
## Proposed change

Documentation added for new Binary Sensors for Flume Notifications

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/77327
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/flume.markdown [hunk 1]
```diff
@@ -27,6 +28,17 @@ The Flume API access is available to Flume Insight users only, which may require
 
 To add `Flume` to your installation, go to **Configuration** >> **Devices & Services** in the UI, click the button with `+` sign and from the list of integrations select **Flume**.
 
+## Notifications
+
+Flume notifications are available via 3 binary sensors. These senors will read the notification state from flume, and if there is an active notification they will report its state. In order to clear the notifications you will need to use your flume app or go to: [https://portal.flumewater.com/notifications]([https://portal.flumewater.com/notifications]) and clear the notification in question.
```
Comment (bdraco) [inline source/_integrations/flume.markdown]:
```suggestion
Flume notifications are available via binary sensors. To clear the notifications, you will need to use your Flume app or go to: [https://portal.flumewater.com/notifications]([https://portal.flumewater.com/notifications]) and clear the notification in question.
```

File: source/_integrations/flume.markdown [hunk 2]
```diff
@@ -27,6 +28,17 @@ The Flume API access is available to Flume Insight users only, which may require
...
+
+The sensors are:
```
Comment (bdraco) [inline source/_integrations/flume.markdown]:
```suggestion
The following notifications are supported:
```

File: source/_integrations/flume.markdown [hunk 3]
```diff
@@ -12,6 +12,7 @@ ha_codeowners:
 ha_domain: flume
 ha_dhcp: true
 ha_platforms:
+  - binary_sensor
   - sensor
 ---
```
Comment (bdraco) [inline source/_integrations/flume.markdown:18]: Please update `ha_category` as well

**Review by bdraco — DISMISSED** [review]
Please see comments above

**Review by frenck — APPROVED** [review]
Thanks, @jeeftor 👍

**Pull request merged**